### ISSUE TITLE
Nexus 7 responsive layout

### ISSUE THREAD

**Issue by x4080** [NativeScript/NativeScript#1311]
I found strange thing using .minWH600.xml layout in nexus 7 :
- If I start the app in portrait, it is using the .minWH600.xml (as it supposed to be I guess)
- but If I start the app in landscape, it is using the regular .xml 

My question is : the responsive layout is depending on the orientation ? If so, I think it is not doing it in Ipad

**Comment by NathanaelA** [comment]
Yeah, the responsive layout system is not well designed.  I cover it in my book a bit; but in a nutshell it loads the layout for the CURRENT view; all future views will continue to use the current view.   So even if you flip from Landscape to Portrait, it will continue to use the Landscape view...

**Comment by x4080** [comment]
Glad that somebody else noticed
